File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// JokeFather SDK exists test

require_once __DIR__ . '/../jokefather_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = JokeFatherSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: lua/test/exists_test.lua
-- JokeFather SDK exists test

local sdk = require("joke-father_sdk")

describe("JokeFatherSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
